.ci/jobs/e2e-testing-mbp.yml

```diff
@@ -5,7 +5,11 @@
     description: Jenkins pipeline for the e2e-testing project
     view: E2E
     project-type: multibranch
-    days-to-keep: 21
+    logrotate:
+      daysToKeep: 30
+      numToKeep: 300
+    number-to-keep: 100
```

Comment:
Did you want these to be the same?

Reply:
Let's started with 100 first, thanks for taking a look!